test/dotnet.Tests/CommandTests/Run/RunFileTests.cs

```diff
@@ -1096,4 +1098,299 @@ public void UpToDate_InvalidOptions()
+                      <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+                      <PropertyGroup>
+                        <OutputType>Exe</OutputType>
+                        <TargetFramework>{ToolsetInfo.CurrentTargetFramework}</TargetFramework>
+                        <ImplicitUsings>enable</ImplicitUsings>
+                        <Nullable>enable</Nullable>
+                      </PropertyGroup>
+
+                      <PropertyGroup>
+                        <EnableDefaultItems>false</EnableDefaultItems>
+                      </PropertyGroup>
+
+                      <PropertyGroup>
+                        <Features>$(Features);FileBasedProgram</Features>
+                      </PropertyGroup>
+
+                      <ItemGroup>
+                        <Compile Include="{programPath}" />
+                      </ItemGroup>
+
+                      <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+
+                      <!--
+                        Override targets which don't work with project files that are not present on disk.
+                        See https://github.com/NuGet/Home/issues/14148.
+                      -->
+
+                      <Target Name="_FilterRestoreGraphProjectInputItems"
+                              DependsOnTargets="_LoadRestoreGraphEntryPoints"
+                              Returns="@(FilteredRestoreGraphProjectInputItems)">
+                        <ItemGroup>
+                          <FilteredRestoreGraphProjectInputItems Include="@(RestoreGraphProjectInputItems)" />
+                        </ItemGroup>
+                      </Target>
+
+                      <Target Name="_GetAllRestoreProjectPathItems"
+                              DependsOnTargets="_FilterRestoreGraphProjectInputItems"
+                              Returns="@(_RestoreProjectPathItems)">
+                        <ItemGroup>
+                          <_RestoreProjectPathItems Include="@(FilteredRestoreGraphProjectInputItems)" />
+                        </ItemGroup>
+                      </Target>
+
+                      <Target Name="_GenerateRestoreGraph"
+                              DependsOnTargets="_FilterRestoreGraphProjectInputItems;_GetAllRestoreProjectPathItems;_GenerateRestoreGraphProjectEntry;_GenerateProjectRestoreGraph"
+                              Returns="@(_RestoreGraphEntry)">
+                        <!-- Output from dependency _GenerateRestoreGraphProjectEntry and _GenerateProjectRestoreGraph -->
+                      </Target>
+
+                    </Project>
+
+                    """)}},"Diagnostics":
+                [{"Location":{
+                "Path":{{ToJson(programPath)}},
+                "Span":{"Start":{"Line":1,"Character":0},"End":{"Line":1,"Character":30}{{nop}}}{{nop}}},
+                "Message":{{ToJson(string.Format(CliCommandStrings.CannotConvertDirective, $"{programPath}:2"))}}}]}
+                """.ReplaceLineEndings(""));
```

Comment:
The JSON output is always on a single line?

Reply:
Yes